Makefile

```diff
@@ -281,7 +312,7 @@ providerconfig-e2e-nopublish:
 uptest-local:
 	@$(WARN) "this target is deprecated, please use 'make uptest' instead"
 
-build-provider.%:
+build-provider.%: $(UPSTREAM)
```

Comment:
Is that really necessary? Because we already put something in for `build.init`. I don't mind leaving it like this. I'm just trying to make sure I understand it correctly.

Reply:
I think you're right, it's probably unnecessary. I probably wrote this change before I figured out the build.init one, and just forgot to remove it.

Reply:
But I think my preference is to leave it here for clarity.